jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/executor/PolicyExecutor.java

```diff
@@ -94,7 +95,8 @@ public PolicyExecutor (String jsonString) {
             source = jsonNode.get(PacmanSdkConstants.SOURCE).asText();
             targetType = jsonNode.get(PacmanSdkConstants.TARGET_TYPE).asText();
         } catch (Exception e) {
-            logger.error(DATA_ALERT_ERROR_STRING + JOB_NAME  + "error in getting Policy arguments  ->"+ e);
+            logger.error(DATA_ALERT_ERROR_STRING + CommonUtils.buildPolicyUUIDFromJson(jsonString) + ERROR_MESSAGE +

```

Comment:
These log statements are very complex to follow for the next dev. What can be done to get rid of opening and closing \"? I strongly suggest that you convert the jsonString to a proper json and use it properly.

Reply:
as per our discussion, the changes are moved to logback.xml